src/MusicStore/config.json

```diff
@@ -1,7 +1,8 @@
 ﻿{
-    "AppSettings": {
-        "SiteTitle": "ASP.NET MVC Music Store"
-    },
+  "AppSettings": {
+    "SiteTitle": "ASP.NET MVC Music Store",
+    "CacheTimeout": "0"
```

Comment:
For the first commit add the original timeout. 
That way you might be able to submit the PR to dev.